miniapps/performance/ex1.cpp

```diff
@@ -13,6 +13,15 @@
 //               ex1 -m ../../data/ball-nurbs.mesh -std  -asm -pc ho  -sc
 //               ex1 -m ../../data/pipe-nurbs.mesh -perf -mf  -pc lor
 //               ex1 -m ../../data/pipe-nurbs.mesh -std  -asm -pc ho  -sc
+//               ex1_2d -m ../../data/star.mesh -perf -mf  -pc lor
+//               ex1_2d -m ../../data/star.mesh -perf -asm -pc ho
+//               ex1_2d -m ../../data/star.mesh -perf -asm -pc ho -sc
+//               ex1_2d -m ../../data/star.mesh -std  -asm -pc ho
+//               ex1_2d -m ../../data/star.mesh -std  -asm -pc ho -sc
+//               ex1_2d -m ../../data/amr-quad.mesh -perf -asm -pc ho -sc
+//               ex1_2d -m ../../data/amr-quad.mesh -std  -asm -pc ho -sc
+//               ex1_2d -m ../../data/disc-nurbs.mesh -perf -asm -pc ho  -sc
+//               ex1_2d -m ../../data/disc-nurbs.mesh -std  -asm -pc ho  -sc
```

Comment:
The scripts that extract the sample runs may need to be updated to support this. Currently, the assumption is that the executable name is the same as the source name with the suffix, `.cpp`, removed.

Reply:
@v-dobrev, what do you think about something [like this](https://gist.github.com/pazner/0c09211b6b506be545e3d6fec8646b07) for including 2D and 3D in the same executable using a template?

Reply:
I think this looks good. Thanks!

Reply:
Ok, great. I just pushed the changes.